contents/blog/introducing-housewatch.md

```diff
@@ -0,0 +1,114 @@
+
+We’ve also built a lot of systems and processes for managing clusters. These include:
+
+- Tracking metrics via Grafana
+- Querying via Metabase
+- Running operations on nodes through `ssh`
+- Managing async migrations with [custom-built tools](/blog/async-migrations)
+
+To share our expertise, formalize these tools, and centralize them in one place, Li, Yakko, and Tim built the first version of HouseWatch at our [Aruba offsite](/blog/aruba-hackathon). We felt it would be useful to us, and support other ClickHouse users as well.
+
+## Features
+
+HouseWatch provides multiple battle-tested tools for monitoring and managing ClickHouse.
+
+### Query performance and analysis
+
+To help understand the performance of all the queries on your ClickHouse clusters, we list queries, with emphasis placed on active and slow queries. 
+
+Each includes metrics on average run time, calls per minute, percentage of all IOPs, total IOPs, and percentage of run time. It also includes details on the query itself and the `EXPLAIN` statement. From this, you can monitor and improve important, slow, or high-stress queries.
+
+![Slow](../images/blog/introducing-housewatch/slow.png)
+
+### Schema stats
+
+HouseWatch provides stats for all the tables for your cluster and lets you dive into the details for each of them. For each table, see columns, parts, compressed and uncompressed disk space sizes, disk usage, and more.
+
+![Schema](../images/blog/introducing-housewatch/table.png)
+
+### Query editing and benchmarking
+
+With the size of data ClickHouse stores and processes, optimizing your queries is critical. To help you do this, you can run and edit queries in HouseWatch, as well as test and compare them.
+
+You can run queries on your ClickHouse cluster and receive results visualized in HouseWatch. For query comparisons, you can write two queries, run them, and then get metrics on performance across the duration, read bytes, CPU usage, memory usage, and more. 
+
+![Compare](../images/blog/introducing-housewatch/compare.png)
+
+We’ve also built a natural language query editor that uses GPT to create ClickHouse queries based on the table and instructions you provide.
+
+![AI](../images/blog/introducing-housewatch/ai.png)
+
+### Logs and errors
+
+Like any good monitoring tool, HouseWatch provides access to logs and errors from ClickHouse. For both, you can search for specific ones, as well as see the number of and most recent occurrences.
+
+![Errors](../images/blog/introducing-housewatch/errors.png)
+
+### Operations
+
+Last but not least is our operations tool. This enables you to run and monitor long-running operations like migrations or SQL commands. You can monitor their status, pause or stop them, and retry them if they fail. Failures come with automatic rollbacks as well.
```

Comment:
I'd mention here again that this is inspired by our async migrations tool which we've been using in production for over a year.

Maybe on one of them we link to the docs and the other to the blog? Or one can be without a link

Reply:
Got it